content/_code-samples/get-started/py/base.py

```diff
@@ -0,0 +1,16 @@
+import asyncio
```

Comment:
I dont think this file is necessary.

Reply:
Same thing, following the JavaScript example as closely as possible, which has a very basic code structure overview.
It would be difficult to have similar code "tabs" without that file.